hub/columns/miscellaneous/oob.tex

```diff
@@ -1,6 +1,6 @@
 \begin{enumerate}[resume]
 	\item $\miscOobInst$:
 		\oobMod{} instruction column; see \ob{TODO: add reference to the \oobMod{} module};
-	\item $\miscOobDataCol{k}$, $k=1, 2, \dots, 9$:
+	\item $\miscOobDataCol{k}$, $k=1, 2, \dots, 10$:
```

Comment:
Extra `misc/OOB_DATA` column. Used purely for the extra field in the updated `OOB_INST_CREATE` instruction.